src/api/PDFFont.ts

```diff
@@ -7,6 +7,18 @@ import {
   StandardFontEmbedder,
 } from 'src/core';
 import { assertIs } from 'src/utils';
+import { BoundingBox } from 'src/types/fontkit';
+
+export interface FontMetrics {
+  /** The font's ascender */
+  ascent: number | void;
+  /** The font's descender */
+  descent: number | void;
+  /** The font's line gap if known; only works on custom fonts currently */
+  lineGap: number;
+  /** Font bounding box */
+  bbox: BoundingBox,
```

Comment:
Instead of using the `BoundingBox` model from `fontkit`, I think it would be best to use the "rectangle" model that is used throughout pdf-lib. This keeps things consistent. That model is as follows:
```ts
{ x: number, y: number, width: number, height: number }
```

Reply:
Can do.

Reply:
Can I ask why you define `{ x: number; y: number; width: number; height: number } ` all over the code instead of actually making it a named interface that can be reused? it would make using the library much nicer...

Reply:
No particular reason. I wouldn't mind having a `Rectangle` interface.